specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/networkManagerDeployment.json

```diff
@@ -0,0 +1,451 @@
+    "description": "The Microsoft Azure Network Manager API provides a RESTful set of web services that interact with Microsoft Azure Network Manager service to manage your network resources.",
+    "version": "2019-11-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkManagers/{networkManagerName}/deploy": {
+      "parameters": [
+        {
+          "$ref": "./network.json#/parameters/ApiVersionParameter"
+        },
+        {
+          "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+        },
+        {
+          "$ref": "#/parameters/ResourceGroupNameParameter"
+        },
+        {
+          "$ref": "#/parameters/NetworkManagerNameParameter"
+        }
+      ],
+      "post": {
+        "tags": [
+          "Deployments"
+        ],
+        "operationId": "Deployments_Post",
+        "description": "The post security deployment operation post a new Managed Network Deployment",
+        "parameters": [
+          {
+            "name": "deployment",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Deployment"
+            },
+            "description": "Parameters supplied to post a Managed Network deployment"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Post successful. The operation returns the resulting Managed Network Deployment.",
+            "schema": {
+              "$ref": "#/definitions/Deployment"
+            }
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DeploymentsPost": {
+            "$ref": "./examples/NetworkManagerDeploymentsPost.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkManagers/{networkManagerName}/activeDeployments/{locationId}": {
```

Comment:
you can leverage the same deployments resource which was used to create the deployment. The deployment type, or whatever you call it, can have state which reflects whther its active or not. ANd then to get the active ones, you can provide filter support through query parameters. adding a new resource type just for getting the active deployments doesn't make sense.